### ISSUE TITLE
Ephemerides

### ISSUE THREAD

**Issue by Juanlu001** [poliastro/poliastro#4]
Look into https://github.com/brandon-rhodes/python-jplephem

**Comment by Juanlu001** [comment]
It would be nice to modify the mass parameters of the planets when loading an ephemeris, see brandon-rhodes/python-jplephem#9.

**Comment by Juanlu001** [comment]
Other than that, this is how the ephemeris can be loaded using jplephem and astropy:

```
>>> from datetime import datetime
>>> from astropy import time
>>> from jplephem.spk import SPK
>>> kernel = SPK.open('de421.bsp')
>>> t = time.Time(datetime.today())
>>> t.jd
2457117.0397053314
>>> kernel[0, 4].compute(t.jd)
array([  1.54722791e+08,   1.42346197e+08,   6.11033174e+07])
>>> kernel[0, 4].compute(t.jd1, t.jd2)
array([  1.54722791e+08,   1.42346197e+08,   6.11033174e+07])
```

jplephem.spk is very low level so perhaps I will have to build a slightly higher level API.

**Comment by Juanlu001** [comment]
Ideas to handle ephemeris data: https://github.com/jaberg/skdata/wiki/Managing-Data

**Comment by Juanlu001** [comment]
Another idea: spacepy copies required files on import: http://sourceforge.net/p/spacepy/code/ci/master/tree/spacepy/__init__.py

May I provide some kind of script instead?

**Comment by Juanlu001** [comment]
In case I go the script way, now that I have setuptools I can use entry points. See also conda-build: http://conda.pydata.org/docs/build.html

**Comment by Juanlu001** [comment]
Closed on 2404de7dfb :tada:
